.travis.yml

```diff
@@ -1,3 +1,6 @@
+# Copyright 2017 syzkaller project authors. All rights reserved.
```

Comment:
[Non-actionable comment, just a question.]

Is there any benefit to switching to SPDX lines or does it not matter?

E.g.
    # SPDX-License-Identifier: Apache-2.0

instead of the long version currently found everywhere?

Reply:
If it's 1 line vs 2 all else being equal, I am all for 1 line.
But we may also need to consult with internal OSS folks and/or grep other google repos for SPDX.